aleph/search/query.py

```diff
@@ -217,18 +216,12 @@ def get_sort(self):
         return list(reversed(sort_fields))
 
     def get_highlight(self):
-        if not self.parser.highlight:
-            return {}
```

Comment:
Is there a particular reason to remove the ability to explicitly enable/disable highlighting? Especially when using the Aleph API directly, it’s often not necessary to return highlights, but computing highlights can consume lots of resources. For example when streaming thousands of entities for offline processing, this could add a lot of unnecessary load on the server.